src/main/resources/io/jenkins/plugins/PurgeBuildQueueAction/action.jelly

```diff
@@ -24,22 +24,6 @@
 
 <?jelly escape-by-default='false'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:f="/lib/form">
-  <j:getStatic var="permission" className="jenkins.model.Jenkins" field="ADMINISTER"/>
-  <l:layout permission="${permission}" title="${%Purge.Build.Queue.Plugin.Title}">
-    <l:side-panel>
-      <l:tasks>
-        <l:task icon="icon-up icon-md" href="${rootURL}/" title="${%Back to Dashboard}" />
-      </l:tasks>
-    </l:side-panel>
-    <l:main-panel>
-      <h1>${%Purge.Build.Queue.Plugin.Title}</h1>
-      <h2>${%Purge.Build.Queue.Plugin.Description}</h2>
-      <br/>
-      <f:form name="purge" action="purge" method="post">
-        <f:block>
-          <input type="submit" value="${%Purge.Build.Queue.Plugin.Submit.Name}" />
-        </f:block>
-      </f:form>
-    </l:main-panel>
-  </l:layout>
+  <l:task icon="images/svgs/edit-delete.svg" confirmationMessage="${%Purge the build queue?}" href="purge-build-queue/purge/" post="true" requiresConfirmation="true" title="${%Purge Build Queue}" permission="${it.ADMINISTER}">
```

Comment:
I'd recommend to use the icon API instead of relying on a path that may be subject to changes in the future, considering it's not API.

Reply:
Done